src/ValidatorVisitor.php

```diff
@@ -358,6 +362,10 @@ public function leaveNode(Node $node){
                     } else {
                         $this->sandbox->validationError("Sandboxed code attempted use invalid namespace or alias!", Error::DEFINE_ALIAS_ERROR, $node);
                     }
+                    if ($this->sandbox->isWhitelistedInterface($use->alias))
```

Comment:
"Coding style":  if ($this->sandbox->isWhitelistedInterface($use->alias)) **{** ;)